src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/PlotterControls.java

```diff
@@ -93,12 +107,7 @@ private JPanel getConnectPanel() {
 		Border border = BorderFactory.createTitledBorder(Translator.get("PlotterControls.ConnectControls"));
 		panel.setBorder(border);
 		panel.add(chooseConnection);
-		chooseConnection.addListener(e -> {
-			switch (e.flag) {
-				case NetworkSessionEvent.CONNECTION_OPENED -> updateButtonStatusOnConnect();
```

Comment:
no `break;`?

Reply:
this is a new case syntax introduced in java 14 : https://betterprogramming.pub/a-look-at-the-new-switch-expressions-in-java-14-ed209c802ba0